.ci/docs

```diff
@@ -4,12 +4,6 @@ def salt_target_branch = '2017.7.9'
 
 properties([
     buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '', numToKeepStr: '10')),
-    [
-        $class: 'ScannerJobProperty', doNotScan: false
-    ],
-    [
-        $class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false
```

Comment:
It works fine removing this line? are rebuilds allowed by default?

Reply:
yes to both Q's

Reply:
cool, sounds good to me.